pkgs/servers/wesher/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, buildGoModule, fetchFromGitHub }:
```

Comment:
I would prefer for these to be separated by newlines, that makes future diffs a lot easier to read.
```suggestion
{ lib
, buildGoModule
, fetchFromGitHub
}:
```

Reply:
I definitely agree with this, changes in https://github.com/NixOS/nixpkgs/pull/194169/commits/2f737c64af82f1e0983e8a619711fe1c8e28d8cf